Client/src/Unified/Services/Submitter/Service.cs

```diff
@@ -495,8 +480,8 @@ private void ResultTask()
   /// <param name="payloads">Th armonikPayload to pass with Function name and serialized arguments</param>
   /// <param name="handler">The handler callBack for Error and response</param>
   /// <returns>Return the taskId</returns>
-  public IEnumerable<string> SubmitTasks(IEnumerable<ArmonikPayload> payloads,
-                                         IServiceInvocationHandler   handler)
+  private IEnumerable<string> SubmitTasks(IEnumerable<ArmonikPayload> payloads,
+                                          IServiceInvocationHandler   handler)
```

Comment:
Please move this function in SessionService. It's another signature with ArmonikPayload

Reply:
Done